builder.go

```diff
@@ -61,6 +62,15 @@ func (logExecutor) EnsureContainerPath(path string) error {
 	return nil
 }
 
+func (logExecutor) EnsureContainerPathAs(path, user string, mode *os.FileMode) error {
+	if mode != nil {
+		log.Printf("ENSURE %s AS %q with MODE=%q", path, user, *mode)
```

Comment:
Is this debug code?

Reply:
The `logExecutor` is used by tests.

Reply:
no, this is the logExecutor that just logs what is happening